### PR TITLE
feat: close prs when unassigned

### LINKED ISSUE #379: Close Pull Request When `Unassigned`
In order to keep our pull requests neat and tidy, the bot should automatically close them out when the bounty hunter is disqualified for inactivity on the bounty. Technically we can inject this behavior on top of the `/unassign` command I believe, that way it will happen both during the disqualification, or if the bounty hunter gives up the bounty by invoking the command.

> I think that makes sense to close the pull requests when unassigned. Perhaps we should also have the bot do that.

_Originally posted by @pavlovcik in https://github.com/ubiquity/ubiquity-dollar/issues/634#issuecomment-1575532629_

### PR DESCRIPTION
**Pull request by wannacfuture**
Resolves #379 
Test successful! 
You can see the result on https://github.com/wannacfuture/Battleship/issues/23
There, when the user unassign, the linked pull requests of him are closed and the comment is added.

### CODE REVIEW

File: src/helpers/issue.ts [hunk 1]
```diff
@@ -338,6 +354,22 @@ export const getAssignedIssues = async (username: string) => {
   return assigned_issues;
 };
 
+export const getOpenedPullRequestsForAnIssue = async (issueNumber: number, userName: string) => {
+  const pulls = await getOpenedPullRequests(userName);
+
+  return pulls.filter((pull) => {
+    if (pull.draft) return false;
```
Comment (rndquu) [inline src/helpers/issue.ts:361]: Shouldn't we also close draft PRs?

Comment (wannacfuture) [reply src/helpers/issue.ts]: Okay. fixed.

**Comment by wannacfuture** [comment]
@Draeieg , Seems I need your approve about QA. Thanks.

**Comment by wannacfuture** [comment]
@0xcodercrane , seems ready to merge. 🤗

**Comment by pavlovcik** [comment]
> Please react with 👍 if the linked QA looks good. @Draeieg

In the future, request a review from Draeieg and then they can add an official approval.

**Pull request merged**